### ISSUE TITLE
GUI result does not show anythin

### ISSUE THREAD

**Issue by phamquandung** [ToniRV/NeRF-SLAM#17]
Hi contributors @ToniRV, @jrpowers,

I build the code successfully. However, when I try to run the code as python ./examples/slam_demo.py --dataset_dir=./datasets/Replica/office0 --dataset_name=nerf --buffer=100 --slam --parallel_run --img_stride=2 --fusion='nerf' --gui. I could not see any gui result. I wonder how to solve this problem. Before that, I also saw an error. Does it matter? 
![image](https://user-images.githubusercontent.com/28943896/210482893-8bd3ff6c-074c-4978-9b85-6602042bbb8e.png)
![image](https://user-images.githubusercontent.com/28943896/210483103-09bf79eb-a266-4f2f-9a81-d29c1d4673b9.png)

**Comment by bkhanal-11** [comment]
Ther is a new PR #12 with this isse and others, I was also facing the same issue but using this [fork](https://github.com/jrpowers/NeRF-SLAM) worked for me.

**Comment by ToniRV** [comment]
Yes, that seems like the instant-ngp is the one from nvidia instead of my fork + branch `feature/nerf-slam'

**Comment by songlin** [comment]
@bkhanal-11 Hi, can you tell me which and how many GPUs you used to run this project?

**Comment by bkhanal-11** [comment]
@songlin I am using 2x NVIDIA RTX 3090.

**Comment by songlin** [comment]
@bkhanal-11 Thanks for your information. I managed to run it on 1 NVIDIA RTX 3090.

**Comment by qungvinh1280** [comment]
> Hi contributors @ToniRV, @jrpowers,
> 
> I build the code successfully. However, when I try to run the code as python ./examples/slam_demo.py --dataset_dir=./datasets/Replica/office0 --dataset_name=nerf --buffer=100 --slam --parallel_run --img_stride=2 --fusion='nerf' --gui. I could not see any gui result. I wonder how to solve this problem. Before that, I also saw an error. Does it matter? ![image](https://user-images.githubusercontent.com/28943896/210482893-8bd3ff6c-074c-4978-9b85-6602042bbb8e.png) ![image](https://user-images.githubusercontent.com/28943896/210483103-09bf79eb-a266-4f2f-9a81-d29c1d4673b9.png)
> 
> And when I try to running with fusion method sigma there is an error as below figure ![image](https://user-images.githubusercontent.com/28943896/210484990-e662f4c6-8c9b-4bd9-8438-fa4b7a8a803d.png)

Tôi cũng gặp vấn đề như bn không pít bn đã khắc phục được nó chưa?? Nếu bn đã khắc phục được thì bn có thể chia sẻ cho t cách khắc phục được không ??
